### DISCUSSION TITLE
How to uninstall Stash from my PC and install it again?

### DISCUSSION THREAD

**Discussion by Ezgril** [stashapp/stash#3640] [Q&A]
Well after trying to install Stash on my PC and getting errors i tried to delete everything and reinstall but now it doesn't work. I am getting the following error:

Stash - Fatal Error
Error: Panic: getting database schema version: unable to open database file

Any idea how I can fix it?

Thank you.

**Comment by MrX292** [comment]
There is probably still a old config.yml file which has a database path in it which no longer exist

**Reply by Ezgril** [reply]
Thanks for the reply

I have deleted everything including the config.yml file and the problem persists. I do something wrong?
